api_collections/booking_collection.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+import json
+from Automation_tests.api_collections.base_api import BaseAPI
+from Automation_tests.api_collections.config_file import USERNAME, PASSWORD
+
+
+class BookingCollection(BaseAPI):
+    def __init__(self):
+        super().__init__()
+        self._booking_url = "booking"
+        self._token_url = "auth"
+        self._update_url = "booking/787"
+
+    def get_token(self):
```

Comment:
why does the get_tocken  in BookingCollection?

Reply:
Fixed